notebooks/inference_notebook.ipynb

```diff
@@ -0,0 +1,287 @@
+{
```

Comment:
<p>Can you remove the extra column "Unnamed"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/operate-first-data-science-community/pull/59/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>